### DISCUSSION TITLE
How to switch between UserControl using SplitView of Avalonia

### DISCUSSION THREAD

**Discussion by Daniil-Kolbasov** [AvaloniaUI/Avalonia#12521] [General]
I want to make a similar NavigationView from UWP, but I can't figure out how to switch between pages in SplitView.Content

Here's how it should look like:
![NavigationView](https://github.com/AvaloniaUI/Avalonia/assets/34882813/3a198472-a1c6-4362-b788-0a3100a19f79)

**Comment by robloo** [comment]
I highly recommend using the NavigationView ported from WinUI in FluentAvalonia https://github.com/amwx/FluentAvalonia. Note that will also provide the Fluent v2 theme for all controls -- so you have to want that as well.

For switching content it's as easy as changing the controls in code-behind. Or changing visibility in XAML based on what you want to show. Just put a Panel or Grid parent and you can have multiple children.
